src/pages/index.js

```diff
@@ -66,53 +61,43 @@ const IndexPage = ({ data, pageContext, location }) => {
           >
             <div className={styles.introText}>
               <p>
-                New Relic Instant Observability (I/O) helps you monitor your
-                stack like a pro, without the burden of manual set up. Explore
-                an open source ecosystem of 500+ quickstarts—out-of-the-box
-                bundles of integrations, dashboards, and alerts—created by
-                observability experts around the world, vetted by New Relic, and
-                ready for you to install with one click.
-                <br />
-                <br />
-                Didn't find what you're looking for? You're always{' '}
-                <a href="https://github.com/newrelic/newrelic-quickstarts">
-                  welcome to add contributions
-                </a>{' '}
-                or improve existing quickstarts to help others instantly monitor
-                their services.
+                New Relic is an observability platform that helps you accurately
+                capture data critical to the success of your team. Whether
```

Comment:
```suggestion
                New Relic is an observability platform that helps you
                capture data critical to the success of your team. Whether
```

Reply:
Not sure this first sentence is the best sell of the platform.

Reply:
Yeah, it's a bit of a hodge podge from existing content (blog post about DevOps/New Relic + Welcome to NR doc). Wanted it to be specific to what a DevOps team might need and guide them to the docs site for their answers